core/kernel/pathname_spec.rb

```diff
@@ -0,0 +1,26 @@
+require_relative '../../spec_helper'
+require 'pathname'
+
+describe "Kernel#Pathname" do
+  it "is a public method" do
+    Kernel.public_methods.should include(:Pathname)
```

Comment:
This should be `Kernel.should have_private_instance_method(:Pathname)` since the spec is about Kernel#Pathname.

There is also `Kernel.Pathname`, which we could test in a separate `it` with `Kernel.should have_method(:Pathname)`.